tools/build_rules/closure/closure_js_binary.bzl

```diff
@@ -46,6 +48,21 @@ _COMPILATION_LEVELS = {
   "advanced": ["--compilation_level=ADVANCED"]
 }
 
+_LANGUAGE_LEVELS = {
```

Comment:
Nit: These aren't "levels". How about naming it _LANGUAGES _SUPPORTED_LANGUAGES?